app/models/mixins/inter_region_api_method_relay.rb

```diff
@@ -39,6 +39,8 @@ def api_relay_class_method(method, action = method)
       define_method(method) do |*meth_args, &meth_block|
         record_id, api_args = yield(*meth_args)
 
+        super if record_id.nil?
```

Comment:
I would just add this to the conditional below. It's the same call to super (as per spec failures)